RSDKUtils/StandardHeaders/URLRequest+StandardHeaders.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public extension URLRequest {
```

Comment:
Using extension functions is a nice idea. Maybe something like this will work for Android too.